JSDemos/Demos/List/ListEditingAndAPI/description.md

```diff
@@ -1 +1,26 @@
-The **List** is&nbsp;a&nbsp;UI component that represents a&nbsp;collection of&nbsp;items in&nbsp;a&nbsp;scrollable list. This demo displays different delete modes and shows how you can handle the **selectionChanged** event.
\ No newline at end of file
+The [List](/Documentation/ApiReference/UI_Widgets/dxList/) displays a collection of items as a scrollable list. To create a list, pass an array of items to **dataSource**. 
+
+This demo illustrates deletion. The UI consists of the **List** and the "Options" section. 
+
+To allow users delete items, set the [allowItemDeleting](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/#allowItemDeleting) property to *true*. Use the checkbox in the "Options" section to see how it works.
```

Comment:
I suggest removing the sentence about the UI because it seems unnecessary if we don't talk about elements in the Options section.

```suggestion
This demo illustrates item deletion. To allow users delete items, set the [allowItemDeleting](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/#allowItemDeleting) property to **true**. Click the Allow deleting checkbox under the List to see how this property works.
```